src/Magnum/Audio/CMakeLists.txt

```diff
@@ -45,6 +45,11 @@ set(MagnumAudio_HEADERS
 
     visibility.h)
 
+if(WITH_SCENEGRAPH)
```

Comment:
Yup, I wanted to fix that for DebugTools, if you enable them, they bring everything else and a kitchen sink now :)